packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -87,7 +87,9 @@ export const Dropdown = forwardRef(
             isDisabled,
             renderOnClickPosition,
             addon,
-            alignMenu = 'left',
+            alignMenu = 'bottom-left',
```

Comment:
first is position from the toggle icon, second alignment on the dropdown itself related to this icon

![image](https://github.com/trezor/trezor-suite/assets/858321/3675773d-2a84-4a15-bfa9-3d71d775b110)

![image](https://github.com/trezor/trezor-suite/assets/858321/2c4a5de3-4b1f-4adc-b1ba-f295475f4d39)

![image](https://github.com/trezor/trezor-suite/assets/858321/180c1e90-13a6-461d-9cf3-73be506ee7a8)

etc.